### PR TITLE
Set guest properly after load in JUser

### PR DESCRIPTION
**Pull request by stefanneculai**
This is a fix for issue #1427

### CODE REVIEW

File: libraries/joomla/user/user.php [hunk 1]
```diff
@@ -828,6 +831,16 @@ public function load($id)
 		// Assuming all is well at this point lets bind the data
```
Comment (elinw) [inline libraries/joomla/user/user.php]: Would you make this "let's" not "lets" please while you're updating tis file.

Comment (stefanneculai) [inline libraries/joomla/user/user.php]: Done.

**Comment by elinw** [comment]
Also would you change the title to be descriptive?

**Comment by stefanneculai** [comment]
Any thoughts about this?

**Pull request merged**